docker/test_jupyter.yaml

```diff
@@ -48,6 +48,17 @@ services:
       - ./models/:/models/
     command: --model_path /models/ --model_name resnet --port 9000 --shape auto --grpc_bind_address 127.0.0.1
 
+  guardian_container:
+    image: pdo_contracts:${CONTRACTS_VERSION}
+    container_name: guardian_container
+    depends_on:
+      - services_container
+      - openvino_container
+    network_mode: "host"
+    volumes:
+      - ./xfer/:/project/pdo/xfer/
+    entrypoint: /project/pdo/tools/run_guardian_tests.sh -i localhost -l http://127.0.0.1:6600
+
```

Comment:
because the configuration of the guardian is unique. the openvino service itself is not a general service because we've loaded a specific model into it.

Reply:
the two scripts, though, are virtually the same and the wait-for-ledger-keys loop could be useful even for `start_...` why not have just one script and use `--mode` as for other scripts to differentiate the site.toml handling?